CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- `iterator.find_map` added
```

Comment:
Could you write this in the style used for the other entries? With a `.` too. Thank you.

Reply:
:heavy_check_mark:

Reply:
Sorry, this still isn't written in the same format all the other additions use.